src/core/matter/mod.rs

```diff
@@ -149,25 +145,22 @@ impl Matter {
             let cs = sizage_.hs + sizage_.ss;
             if cs % 4 != 0 {
                 return Err(Box::new(Error::InvalidCodeSize(format!(
-                    "whole code size not multiple of 4 for variable length material: cs = [{}]",
-                    cs
+                    "whole code size not multiple of 4 for variable length material: cs = [{cs}]",
                 ))));
             };
 
             const SIXTY_FOUR: u32 = 64;
             if SIXTY_FOUR.pow(sizage_.ss) - 1 < size {
                 return Err(Box::new(Error::InvalidVarSize(format!(
-                    "invalid size for code: size = [{}], code = [{}]",
-                    size, code
+                    "invalid size for code: size = [{size}], code = [{code}]",
                 ))));
             }
 
             let both = format!("{}{}", code, util::u32_to_b64(size, sizage_.ss as usize));
 
             if both.len() % 4 != ps - sizage_.ls as usize {
                 return Err(Box::new(Error::InvalidCodeSize(format!(
-                    "invalid code for converted raw pad size: code = [{}], pad size = [{}]",
-                    both, ps
+                    "invalid code for converted raw pad size: code = [{both}], pad size = [{ps}]",
```

Comment:
are the []'s just for style or am I missing something? I'd like to be consistent in the Indexer work

Reply:
It is, yeah. For parsability, I suppose, but the more I thought about this the more I realized we probably want to return structured json since that's what a server will likely be logging. Thoughts?